Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -439,6 +445,16 @@ void getSupportedTypes(HashSet<String, ASCIICaseInsensitiveHash>& types) const f
     m_visible = visible;
     if (m_visible)
         acceleratedRenderingStateChanged();
+
+#if PLATFORM(VISION)
+    if ([m_sampleBufferDisplayLayer respondsToSelector:@selector(setSTSLabel:)]) {
```

Comment:
Nit: I'd leave a comment here noting that the `-respondsToSelector:` check can be removed after 114048644 is resolved.

Reply:
Removed the respondsToSelector now that we are using SPI instead 👍